apps/desktop/desktop_native/autotype/src/windows.rs

```diff
@@ -56,6 +60,44 @@ pub fn type_input(input: Vec<u16>) -> Result<(), ()> {
     send_input(keyboard_inputs)
 }
 
+/// Converts a valid shortcut key to an "up" keyboard input.
+///
+/// `input` must be a valid shortcut key: Control, Alt, Super, Shift, letters [a-z][A-Z]
+fn convert_shortcut_key_to_up_input(key: String) -> Result<INPUT, ()> {
+    const SHIFT_KEY: u8 = 0x10;
+    const SHIFT_KEY_STR: &str = "Shift";
+    const CONTROL_KEY: u8 = 0x11;
+    const CONTROL_KEY_STR: &str = "Control";
+    const ALT_KEY: u8 = 0x12;
+    const ALT_KEY_STR: &str = "Alt";
+    const LEFT_WINDOWS_KEY: u8 = 0x5B;
+    const LEFT_WINDOWS_KEY_STR: &str = "Super";
+
+    Ok(match key.as_str() {
+        SHIFT_KEY_STR => build_virtual_key_input(InputKeyPress::Up, SHIFT_KEY),
+        CONTROL_KEY_STR => build_virtual_key_input(InputKeyPress::Up, CONTROL_KEY),
+        ALT_KEY_STR => build_virtual_key_input(InputKeyPress::Up, ALT_KEY),
+        LEFT_WINDOWS_KEY_STR => build_virtual_key_input(InputKeyPress::Up, LEFT_WINDOWS_KEY),
+        _ => build_unicode_input(InputKeyPress::Up, encode_utf16_alphabetic_hotkey(key)?),
+    })
+}
+
+/// Given a letter that is a String, get the utf16 encoded
+/// decimal version of the letter as long as it meets the
+/// [a-z][A-Z] restriction.
+fn encode_utf16_alphabetic_hotkey(letter: String) -> Result<u16, ()> {
+    if letter.len() != 1 {
+        return Err(());
+    }
+
+    let c = letter.chars().next().expect("we shouldn't reach this");
```

Comment:
Expect is an assertion so the wording can reflect what expectation was invalidated.

```suggestion
    let c = letter.chars().next().expect("letter is size 1");
```